rest-endpoint/src/main/java/com/cognifide/aet/rest/BasicDataServlet.java

```diff
@@ -82,7 +82,7 @@ void unregister(String servletPath) {
     setHttpService(null);
   }
 
-  boolean isValidName(String suiteName) {
+  public static boolean isValidName(String suiteName) {
```

Comment:
If those methods are used outside the servlet scope please extract them.
`BasicDataServlet` purpose shouldn't be mixed with Service/Helper that provide some common functionalities.